lessons/bn/basics/collections.md

```diff
@@ -79,14 +82,16 @@ iex> tail
 [:pie, "Apple"]
 ```
 
-## টাপ্ল 
+## টাপ্ল
+
+টাপ্ল লিস্টের মতই একটি কালেকশন কিন্তু টাপলে ডাটা একাধারে সংরক্ষিত থাকে। এর মানে হল, টাপলের দৈর্ঘ্য জানা যায় অনেক দ্রুত কিন্তু টাপলের মান চেঞ্জ করা একটু মুসকিল হয়ে পড়ে। টাপ্লকে মডিফাই করতে হলে পুরা ডাটাকে মেমরিতে রাখা লাগে তাই টাপল মডিফিকেশন না করাই ভাল।
 
-টাপ্ল লিস্টের মতই একটি কালেকশন কিন্তু টাপলে ডাটা একাধারে সংরক্ষিত থাকে। এর মানে হল, টাপলের দৈর্ঘ্য জানা যায় অনেক দ্রুত। কিন্তু টাপ্লকে মডিফাই করতে হলে পুরা ডাটাকে মেমরিতে রাখা লাগে তাই টাপল মডিফিকেশন না করাই ভাল।  
 ```elixir
 iex> {3.14, :pie, "Apple"}
 {3.14, :pie, "Apple"}
 ```
-টাপল প্রায়েই ব্যবহৃত হয় ফাংশনের রীটার্ন ভ্যালু হিসেবে যখন অতিরিক্ত ডাটা প্রয়োজন, এ সম্পর্কে আরও জানা যাবে যখন আমরা প্যাটার্ন ম্যাচিং করব- 
+
+টাপল প্রায়েই ব্যবহৃত হয় ফাংশনের রীটার্ন ভ্যালু হিসেবে যখন অতিরিক্ত ডাটা প্রয়োজন, এ সম্পর্কে আরও জানা যাবে যখন আমরা [প্যাটার্ন ম্যাচিং](/bn/lessons/basics/pattern_matching) করব:
```

Comment:
Another `:` from legacy translations

Reply:
@wasi0013 bhai, I think using ":" in these cases is fine since ঃ is a character from the bengali alphabet, having a special sound, usecase, and not a punctuation mark. Let me know your thoughts in this regard.

Reply:
Thanks, learned something new! My bad, I've been abusing it as a symbol... 
Guess I will have to refactor some of my own translations! 😅